Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SimpleHTTPServer.py", line 235, in <module>
    test()
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SimpleHTTPServer.py", line 231, in test
    BaseHTTPServer.test(HandlerClass, ServerClass)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/BaseHTTPServer.py", line 595, in test
    httpd = ServerClass(server_address, HandlerClass)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SocketServer.py", line 420, in __init__
    self.server_bind()
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SocketServer.py", line 434, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 48] Address already in use
192.168.100.5 [2016-02-04T14:12:33+07:00]  "POST /inform" 404 10
192.168.100.7 [2016-02-04T14:12:41+07:00]  "POST /inform" 404 10
192.168.100.5 [2016-02-04T14:13:04+07:00]  "POST /inform" 404 10
192.168.100.7 [2016-02-04T14:13:11+07:00]  "POST /inform" 404 10
192.168.100.17 [2016-02-04T14:13:29+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:30+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:30+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:30+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:31+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:33+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.5 [2016-02-04T14:13:34+07:00]  "POST /inform" 404 10
192.168.100.17 [2016-02-04T14:13:36+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.7 [2016-02-04T14:13:41+07:00]  "POST /inform" 404 10
192.168.100.17 [2016-02-04T14:13:42+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
192.168.100.17 [2016-02-04T14:13:55+07:00] 192.168.100.2 "GET /cloud-config-master.yml" 404 34
